components/url/lib.rs

```diff
@@ -49,8 +49,8 @@ impl ServoUrl {
 
     // NOTE: These methods return options that are always true temporarily until
     // we special-case some urls to avoid going through rust-url.
```

Comment:
@jdm This was from a time where we thought `ServoUrl` wouldn't always contain an `Url`, will remove it.